internal_worker_base.go

```diff
@@ -107,11 +114,16 @@ func createPollRetryPolicy() backoff.RetryPolicy {
 
 func newBaseWorker(options baseWorkerOptions, logger *zap.Logger) *baseWorker {
 	return &baseWorker{
-		options:     options,
-		shutdownCh:  make(chan struct{}),
-		rateLimiter: common.NewTokenBucket(1000, common.NewRealTimeSource()),
-		retrier:     backoff.NewConcurrentRetrier(pollOperationRetryPolicy),
-		logger:      logger.With(zapcore.Field{Key: tagWorkerType, Type: zapcore.StringType, String: options.workerType}),
+		options:         options,
+		shutdownCh:      make(chan struct{}),
+		pollRateLimiter: common.NewTokenBucket(1000, common.NewRealTimeSource()),
```

Comment:
I think we want to protect our server and randomly put 1000 at first, this is what it was, i didn't change it in this diff.